src/code/audio/sfx_entry_point.asm

```diff
@@ -1,20 +1,20 @@
 ; Entry point for the Sfx sound system
 SoundSystemInit::
-    jp   label_01F_4009                           ; $4000: $C3 $09 $40
+    jp   _SoundSystemInit                           ; $4000: $C3 $09 $40
```

Comment:
please have a look for the comment indentation

Reply:
Thanks for the catch. I probably messed up something in dialog_letters.asm, cause all comment were wrongly indented.

I fixed this.